Field.h

```diff
@@ -125,4 +126,38 @@ class TextField: public Field {
         string getValue() {
             return fieldValue;
         }
+};
+
+class CharField: public Field {
+    
+    private:
+        string fieldValue;
+        int maxLength;
+        
+    public:
+        CharField(string name, int maxLen, string defaultValue="", bool primary=false, bool unique=false, bool nullable=true):
```

Comment:
Here `maxLength` has already been declared as a private variable above (line 135). I think you had the same problem with the `Field` class where the final parameter is set as `bool autoF=false` for the member variable `autoFilled`. We will need to decide what to do with these cases in future since they will probably be quite common:
https://stackoverflow.com/questions/10250016/should-i-use-the-same-name-for-a-member-variable-and-a-function-parameter-in-c